betterproto/__init__.py

```diff
@@ -14,9 +14,9 @@
     Dict,
     Generator,
     Iterable,
-    List,
+    Iterator, List,
     Optional,
-    SupportsBytes,
+    Sequence, SupportsBytes,
```

Comment:
if Sequence used?

Reply:
Doesn't appear to be; must have been an earlier idea; I'll clean that up.